### PR TITLE
 Use to instead of bcc in gem push mails

### PR DESCRIPTION
**Pull request by sonalkr132**
It was sending mails with empty `to` and no `bcc`.

### CODE REVIEW

File: app/mailers/mailer.rb [hunk 1]
```diff
@@ -47,8 +47,10 @@ def gem_pushed(pushed_by_user_id, version_id)
 
     @pushed_by_user = User.find(pushed_by_user_id)
 
-    mail bcc: owners_to_notify,
-         subject: I18n.t("mailer.gem_pushed.subject", gem: @version.to_title,
-           default: "Gem %{gem} pushed to RubyGems.org")
+    owners_to_notify.each do |owner|
+      mail to: owner,
```
Comment (eliotsykes) [inline app/mailers/mailer.rb]: @sonalkr132 I've got a vague recollection that multiple `mail` calls in a mailer method can cause buggy behaviour under some circumstances. 

Its something to do with the way mailers use class methods to invoke these instance methods. I'm afraid I can't remember more than that.

Comment (sonalkr132) [reply app/mailers/mailer.rb]: Thanks for pointing out. I will add a test for it.

Comment (eliotsykes) [reply app/mailers/mailer.rb]: @sonalkr132 Sorry I'm not being very precise here. I suspec this SO answer is hitting the issue: https://stackoverflow.com/a/48134459/67834

IIRC, the mail only sends to the `to:` passed in on the very last `mail` call, i.e. `owners_to_notify.last` will be sent the email, the other owners won't be.

Comment (simi) [reply app/mailers/mailer.rb]: ahh, move the iteration to mailer call instead

**Pull request merged**